packages/nimble-components/src/theme-provider/design-token-names.ts

```diff
@@ -196,6 +177,33 @@ export const tokenNames: { readonly [key in TokenName]: string } = {
     bodyFontWeight: 'body-font-weight',
     bodyFontLineHeight: 'body-font-line-height',
     bodyFallbackFontFamily: 'body-fallback-font-family',
+    bodyEmphasizedFont: 'body-emphasized-font',
+    bodyEmphasizedFontColor: 'body-emphasized-font-color',
+    bodyEmphasizedDisabledFontColor: 'body-emphasized-disabled-font-color',
+    bodyEmphasizedFontFamily: 'body-emphasized-font-family',
+    bodyEmphasizedFontSize: 'body-emphasized-font-size',
+    bodyEmphasizedFontWeight: 'body-emphasized-font-weight',
+    bodyEmphasizedFontLineHeight: 'body-emphasized-font-line-height',
+    bodyEmphasizedFallbackFontFamily: 'body-emphasized-fallback-font-family',
+    bodyPlus1Font: 'body-plus-1-font',
+    bodyPlus1FontColor: 'body-plus-1-font-color',
+    bodyPlus1DisabledFontColor: 'body-plus-1-disabled-font-color',
+    bodyPlus1FontFamily: 'body-plus-1-font-family',
+    bodyPlus1FontSize: 'body-plus-1-font-size',
+    bodyPlus1FontWeight: 'body-plus-1-font-weight',
+    bodyPlus1FontLineHeight: 'body-plus-1-font-line-height',
+    bodyPlus1FallbackFontFamily: 'body-plus-1-fallback-font-family',
+    bodyPlus1EmphasizedFont: 'body-plus-1-emphasized-font',
+    bodyPlus1EmphasizedFontColor: 'body-plus-1-emphasized-font-color',
+    bodyPlus1DisabledEmphasizedFontColor:
```

Comment:
if this is:
element: body-plus-1
states: disabled, emphasized (sort states alphabetically)
type: font-color

Then the naming is accurate. But instead we treat DisabledFontColor like a type (maybe accidentally). Above in this PR we have bodyEmphasizedDisabledFontColor and we have some other tokens like linkActiveProminentDisabledFontColor.

I think what we should do is say DisabledFontColor is it's own type officially.

Proposal:

So we should name this token `bodyPlus1EmphasizedDisabledFontColor` for:
element: body-plus-1
state: emphasized
type: disabled-font-color

And then in the list of token suffixes in this file, **before** `FontColor`, we should add `DisabledFontColor`:

https://github.com/ni/nimble/blob/55ba7aa29788bdeeb90726f2e6b9b56e568a85ba/packages/nimble-components/src/theme-provider/design-token-names.ts#L273

And then in tokens.stories.ts again before `FontColor` we should add `DisabledFontColor` and map to the color template:

https://github.com/ni/nimble/blob/55ba7aa29788bdeeb90726f2e6b9b56e568a85ba/packages/nimble-components/src/theme-provider/tests/tokens.stories.ts#L99

Reply:
Technically that suffix list should also have FallbackFontFamily but we want to stop exposing those as tokens so we can avoid adding it to the list, see https://github.com/ni/nimble/issues/1192#issuecomment-1994851055